Dockerfile

```diff
@@ -6,7 +6,11 @@ ARG PROJECT=samply
 ENV PROJECT=$PROJECT
 
 RUN ["rm", "-fr", "/usr/local/tomcat/webapps"]
-ADD target/connector.war                        /usr/local/tomcat/webapps/ROOT.war
+
+RUN set -x ; \
```

Comment:
Can we please move the introduction of a separate user into its own PR? At the end, tomcat is still executed as root. If we move to a non-root user, we should do it everywhere.

Reply:
Right -- the user was introduced in the Mainzelliste docker container to support Tomcat running as non-root. Seems like the official Tomcat image runs as root anyways. I've just checked: Tomcat extracts everything as root:root, so I've now mimicked that behaviour. PR updated.